apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/ConfigServerEurekaClientConfigure.java

```diff
@@ -0,0 +1,42 @@
+package com.ctrip.framework.apollo.configservice;
+
+import com.ctrip.framework.apollo.biz.eureka.ApolloEurekaClientConfig;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.cloud.context.scope.refresh.RefreshScope;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.event.EventListener;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * @author : kl
+ * After startup, set FetchRegistry to true, refresh eureka client
+ **/
+@Configuration
+@ConditionalOnProperty(value = {"eureka.client.enabled", "apollo.eureka.server.enabled"}, havingValue = "true", matchIfMissing = true)
```

Comment:
As we have changed to solution to use `ApplicationReadyEvent`, then I think we need to activate this as long as `eureka.client.enabled` no matter it is an eureka server or not?
```suggestion
@ConditionalOnProperty(value = {"eureka.client.enabled", "apollo.eureka.server.enabled"}, havingValue = "true", matchIfMissing = true)
```

Reply:
Didn't understand what you mean, shouldn't this only work in the eureka registry mode?

Reply:
I think I understand what you mean, in the latest mode, just use `eureka.client.enabled` to control